### PR TITLE
Fix typos: Github to GitHub

### PR DESCRIPTION
**Pull request by jbampton**

### CODE REVIEW

File: site/developer-tools.html [hunk 1]
```diff
@@ -484,8 +484,8 @@ <h4>Solving a binary incompatibility</h4>
 
 <p>For the problem described above, we might add the following:</p>
 
-<figure class="highlight"><pre><code class="language-scala" data-lang="scala"><span class="c1">// [SPARK-zz][CORE] Fix an issue</span>
-<span class="nv">ProblemFilters</span><span class="o">.</span><span class="py">exclude</span><span class="o">[</span><span class="kt">DirectMissingMethodProblem</span><span class="o">](</span><span class="s">"org.apache.spark.SomeClass.this"</span><span class="o">)</span></code></pre></figure>
+<figure class="highlight"><pre><code class="language-scala" data-lang="scala"><span class="c1">// [SPARK-zz][CORE] Fix an issue
+</span><span class="nv">ProblemFilters</span><span class="o">.</span><span class="py">exclude</span><span class="o">[</span><span class="kt">DirectMissingMethodProblem</span><span class="o">](</span><span class="s">"org.apache.spark.SomeClass.this"</span><span class="o">)</span></code></pre></figure>
```
Comment (maropu) [inline site/developer-tools.html]: Why did you move `</span>` into the next line?

File: site/news/index.html [hunk 2]
```diff
@@ -217,6 +217,7 @@ <h3 class="entry-title"><a href="/news/spark-3.0.0-preview2.html">Preview releas
       <div class="entry-date">December 23, 2019</div>
     </header>
     <div class="entry-content"><p>To enable wide-scale community testing of the upcoming Spark 3.0 release, the Apache Spark community has posted a <a href="https://archive.apache.org/dist/spark/spark-3.0.0-preview2/">Spark 3.0.0 preview2 release</a>. This preview is <b>not a stable release in terms of either API or functionality</b>, but it is meant to give the community early access to try the code that will become Spark 3.0. If you would like to test the release, please download it, and send feedback using either the <a href="https://spark.apache.org/community.html">mailing lists</a> or <a href="https://issues.apache.org/jira/browse/SPARK/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel">JIRA</a>. The documentation is available at the <a href="https://spark.apache.org/docs/3.0.0-preview2/">link</a>.</p>
+
```
Comment (maropu) [inline site/news/index.html]: Why do we need the changes?

File: contributing.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ feedback on any performance or correctness issues found in the newer release.
 <h2>Contributing by Reviewing Changes</h2>
 
 Changes to Spark source code are proposed, reviewed and committed via 
-<a href="https://github.com/apache/spark/pulls">Github pull requests</a> (described later). 
+<a href="https://github.com/apache/spark/pulls">GitHub pull requests</a> (described later).
```
Comment (maropu) [inline contributing.md:46]: Ur, fixing the typos looks fine.

File: site/examples.html [hunk 4]
```diff
@@ -230,11 +230,11 @@ <h3>Word Count</h3>
...
+<figure class="highlight"><pre><code class="language-python" data-lang="python"><span class="n">text_file</span> <span class="o">=</span> <span class="n">sc</span><span class="p">.</span><span class="n">textFile</span><span class="p">(</span><span class="s">"hdfs://..."</span><span class="p">)</span>
+<span class="n">counts</span> <span class="o">=</span> <span class="n">text_file</span><span class="p">.</span><span class="n">flatMap</span><span class="p">(</span><span class="k">lambda</span> <span class="n">line</span><span class="p">:</span> <span class="n">line</span><span class="p">.</span><span class="n">split</span><span class="p">(</span><span class="s">" "</span><span class="p">))</span> \
+             <span class="p">.</span><span class="nb">map</span><span class="p">(</span><span class="k">lambda</span> <span class="n">word</span><span class="p">:</span> <span class="p">(</span><span class="n">word</span><span class="p">,</span> <span class="mi">1</span><span class="p">))</span> \
```
Comment (maropu) [inline site/examples.html]: It seems the changes are different from the PR title:  `Fix case of GitHub`.

**Comment by jbampton** [comment]
Seems  `jekyll build` created the extra changes.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by maropu** [comment]
LGTM

**Comment by maropu** [comment]
Thanks! Merged to asf-site.
